packages/flutter/test/material/flat_button_test.dart

```diff
@@ -333,6 +333,79 @@ void main() {
         paintsExactlyCountTimes(#clipPath, 0),
     );
   });
+
+  testWidgets('FlatButton responds to tap and onLongPress when enabled', (WidgetTester tester) async {
+
+    int pressedCount = 0;
+
+    Widget buildFrame({VoidCallback onPressed, VoidCallback onLongPress}) {
+      return Directionality(
+        textDirection: TextDirection.ltr,
+        child: FlatButton(child: const Text('button'), onPressed: onPressed, onLongPress: onLongPress),
```

Comment:
I prefer breaking function calls with multiple named parameters like here and elsewhere into multiple lines because I find it easier to read, but it's up to you. For example:
```suggestion
        child: FlatButton(
          child: const Text('button'),
          onPressed: onPressed,
          onLongPress: onLongPress,
        ),
```